build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+plugins {
+  id("java-gradle-plugin")
+  id("com.diffplug.spotless") version libs.versions.spotless.gradle.get()
+  id("org.sonarsource.iac.artifactory-configuration")
+  // id("org.sonarsource.text.rule-api")
+  id("org.sonarsource.iac.sonarqube")
+  id("com.diffplug.blowdryer")
```

Comment:
IMO it's not needed

Reply:
I used it to provide spotless with the eclipse-formatter config from URL. Apparently, it was the easiest way to depend on a remote file with some kind of caching and not to download it every time